src/lifecycle/created.js

```diff
@@ -67,7 +78,7 @@ export default function (opts) {
     }
 
     if (propertyDefinitions) {
-      propertiesCreated(this, propertyDefinitions);
+      iniitaliseProperties(this, propertyDefinitions);
```

Comment:
Yeah, this is fixed in #542. The PRs all contain the work of the last one, that's why they're numbered. I wanted to break the work up into chunks.